hwpf/fapi2/include/plat/plat_error.H

```diff
@@ -0,0 +1,46 @@
+/* IBM_PROLOG_BEGIN_TAG                                                   */
```

Comment:
either keep the correct prologue or remove.

Reply:
removed the prolog